editoast/fga/src/client.rs

```diff
@@ -907,10 +906,7 @@ impl PreparedDeletes<'_> {
 
 impl Client {
     fn base_url(&self) -> url::Url {
```

Comment:
```suggestion
    fn base_url(&self) -> &url::Url {
```

we could make it a ref now